### PR TITLE
Schema approved - ready for review

### PR DESCRIPTION
**Pull request by gregoryrichens**
Cassandra schema
Postgres schema
Readme

### CODE REVIEW

**Review by simei-yeh — APPROVED** [review]
looks good! great schema and readme, no substantial comments

File: db/Cassandra/schema.cql [hunk 1]
```diff
@@ -0,0 +1,34 @@
+CREATE KEYSPACE ReviewsDB WITH replication = {
+  'class':'SimpleStrategy',
+  'replication_factor': 3
```
Comment (simei-yeh) [inline db/Cassandra/schema.cql]: if you want to save HD space, you can replicate only 1 time

Comment (gregoryrichens) [reply db/Cassandra/schema.cql]: thanks!

**Review by McC4b3r — COMMENTED** [review]
schema looks good and api routes look intuitive. looking good and should be a strong foundation to move forward

**Pull request merged**